docs/configuration.md

```diff
@@ -1033,3 +1033,67 @@ fail-fast matrices and support is more limited because of that expectation.
     ```
     failFast.set(true)
     ```
+
+
+### additionalFlankConfig
+Allow appending additional config to flank root yaml. This option is useful when you would like to test option
+before it is available on Fladle. Supports both single and multiple properties.
+
+Single property
+
+=== "Groovy"
+```
+additionalFlankConfig = "new-property: true"
+```
```

Comment:
Please add the spacing to all of the documentation so that the mkdocs formatting works.
```suggestion
    ```
    additionalFlankConfig = "new-property: true"
    ```
```

Reply:
fixed